libraries/lib-datahandler/src/main/thrift/users.thrift

```diff
@@ -17,6 +17,7 @@ enum UserGroup {
     USER = 0,
     ADMIN = 1,
     CLEARING_ADMIN = 2,
+    ECC_ADMIN = 3,
```

Comment:
Yes, this isn't nice. I think it'll be ok this time if I change to `ECC_ADMIN = 5`. As long as we don't have this code in production, we break any important database by changing the value. What do you think?